core/http/src/main/java/org/trellisldp/http/core/HttpSession.java

```diff
@@ -83,4 +92,21 @@ public IRI getAgent() {
     public Instant getCreated() {
         return created;
     }
+
+    /**
+     * Create a session from a security context.
+     * @param security the security context
+     * @return the session
+     */
+    public static Session from(final SecurityContext security) {
```

Comment:
Maybe I'm being too picky, but since this method has the potential to create an administrative session, maybe it should be `final`?

Reply:
I thought `static` methods are _ipso facto_ `final`. Or am I mistaken there?

Reply:
No, they can be hidden (not overridden) in subclasses. But like I said, I'm not really worried about it. On second thought, I wouldn't fool with it.